.pre-commit-config.yaml

```diff
@@ -140,7 +140,7 @@ repos:
         pass_filenames: false
         types: [python]
         stages: [manual]
-    -   id: mypy
+    -   id: stubtest
```

Comment:
docs indicated you could just say `stubtest`, but that didn't work without this change.  Now you can run `mypy` and `stubtest` separately as part of type checking